### DISCUSSION TITLE
Betas 11-14 causing stuttering on Logic audio busses

### DISCUSSION THREAD

**Discussion by bluesbop** [apohl79/audiogridder#1055] [Help] [answered]
I had been using 1.2.0 beta 6 for a long time with no issues and finally got around to downloading beta 14. When inserting the plugin on a bus I would get stuttering in the audio, even with no plugins hosted, but individual channels worked fine. I confirmed this by opening some old sessions that worked flawlessly with beta6 and they also stuttered with AG on a bus or the stereo output. 

I installed the previous releases one by one to test and beta 10 is the last release that works reliably on my setup (Logic 10.6.3, Mac OS 10.15.7).

**Comment by apohl79** [comment, marked as answer]
It does not happen to me with b14 but I think it is related to a bug in the buffering code that got fixed lately. 

Try to disable the "Fixed buffer size" setting.

**Reply by bluesbop** [reply]
That fixed it...thank you! I really appreciate the work you're putting into this software, I couldn't mix without it.
